src/All-Movies/All-Movies.css

```diff
@@ -1,11 +1,21 @@
 .movies-container {
-        display: grid;
-        grid-template-columns: 300px 300px 300px 300px 300px 300px 300px 300px;
-        grid-gap: 10px;
-        background-color: #444;
-        border-radius: 5px;
-        padding: 20px;
-        font-size: 150%;
-        height: auto;
-        width: 95vw;
-}
\ No newline at end of file
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
```

Comment:
what do you think of the sizing here does it look good?

Reply:
This looks really good, I pulled the branch down to test it and the grid works great! I especially like the minmax/auto-fit 🤌🏻

Reply:
ty!